### PR TITLE
Adding DPT 10 and DPT 11 

### PR DESCRIPTION
**Pull request by evlas**
Passing timestamp to the DPT10 and DPT11, the correct KNX package is generated

### CODE REVIEW

File: knx.js [hunk 1]
```diff
@@ -207,6 +207,40 @@ module.exports = function (RED) {
                     buf.writeFloatLE(value, 0);
                     value = buf;
                     break;
+                case '10':   //3Byte fron timestamp
```
Comment (aborovsky) [inline knx.js]: typo in `fron` :)

Comment (aborovsky) [inline knx.js]: May i ask you to provide few test cases for both DPT10 and DPT11?
For example:
`1550745501013` -> `Buffer[ 141, 38, 21 ]`

I'll add unit tests on the base of Mocha + ChaiJS

**Review by aborovsky — CHANGES_REQUESTED** [review]
Thank you a lot for PR!
It's great and i like your idea to accept and handle time as timestamp. It's pretty comfort for JS environment.

I've merged concurrent PR: https://github.com/estbeetoo/node-red-contrib-knx/pull/22/files
There implemented DPT10, but incoming data must be a structure: `{ "day": 3, "hours": 11, "minutes": 22, "seconds": 44 }, "dpt": "10" }`

I propose to support both alternatives for DPT10: incoming data may be a structure or just a plain timestamp.

Also, i'd like to ask you to improve knx.html knx-out node's online readme: add DTP10 and DPT11 message format type. As it was done here: [knx.html improvement](https://github.com/estbeetoo/node-red-contrib-knx/commit/208ca95b68f394a217cf6cce757e246db97be1d5#diff-ab0080e7c56d0c8800e08c27a674f53eR94)

Please, improve your PR.

**Comment by evlas** [comment]
Hello,
sorry, I do not know Mocha and ChalJS.
I generated 2 files to test DPT10 and DPT11.

Il giorno gio 21 feb 2019 alle ore 11:46 Alexander Borovsky <
notifications@github.com> ha scritto:

**Comment by evlas** [comment]
Hello,
I have mixed my code with yours

**Comment by aborovsky** [comment]
@evlas thank you for improvements! But there are still unresolved conflicts. May i ask you to resolve them?

**Pull request merged**